src/App.js

```diff
@@ -18,7 +19,10 @@ function App() {
   const [message, setMessage] = useState('');
   const { status } = useAuth();
   const isLoggedIn = status === AuthStatus.LoggedIn;
-  // const navigate = useNavigate();
+  // create an axios instance
+  const instance = axios.create({
+    withCredentials: true,
+  });
```

Comment:
Is this being used, in this file?

Reply:
I don't have this lines in my App.js anymore, I have no idea how still it appears in this branch

Reply:
No it does not. I will remove it